src/index.ts

```diff
@@ -65,6 +65,7 @@ const PROCESSORS = (nftFactories: NftFactory[], metaFactories: MetaFactory[], mu
     ipfsAudioPinner,
     ipfsArtworkPinner,
     errorProcessor,
+    errorAndMetadataResetProcessor,
```

Comment:
Is it worthwhile making this run before the `metaFactoryProcessors` so that it is run only the next time `yarn start` is triggered? That way between run 1 and run 2 we won't lose track of the `metadata` on any errors.